### PR TITLE
Adding GPU Driver logic for deployment scripts

### PR DESCRIPTION
**Pull request by tylergibson**
Needed to ensure we install the correct driver for the correct version of Windows Server and correct video card series.

### CODE REVIEW

File: Samples/Server/Directx-Multithreaded/Deploy/deploy-rendering-server.ps1 [hunk 1]
```diff
@@ -1,6 +1,36 @@
-$gpuDriverVersion = "385.08" 
-$gpuDriverSuffix = "-tesla-desktop-winserver2016-international-whql.exe"
-$gpuDriverBaseUri = "http://us.download.nvidia.com/Windows/Quadro_Certified/"
+if($args.Length -gt 0)
```
Comment (bengreenier) [inline Samples/Server/Directx-Multithreaded/Deploy/deploy-rendering-server.ps1]: `-gt 1` now, right?

Comment (tylergibson) [reply Samples/Server/Directx-Multithreaded/Deploy/deploy-rendering-server.ps1]: I added this as a check for anyone who may use the deploy script directly, outside of VS.  We actually never used the arg we were passing previously :)

Comment (bengreenier) [reply Samples/Server/Directx-Multithreaded/Deploy/deploy-rendering-server.ps1]: makes sense! let's remove the check of update the condition, but then this looks great! 🔥 🚀

Comment (tylergibson) [reply Samples/Server/Directx-Multithreaded/Deploy/deploy-rendering-server.ps1]: Ahh, now I understand what your previous comment was.  Totally make sense, will update.

**Pull request merged**